roles/requirements.yml

```diff
@@ -2,4 +2,4 @@
 - name: cloudalchemy.prometheus
 - name: cloudalchemy.grafana
 - name: geerlingguy.certbot
-- name: nginxinc.nginx
\ No newline at end of file
+- name: community.general
```

Comment:
What do we need that for?

Reply:
For the [npm module](https://github.com/juwi/palava-ansible/pull/12/files#diff-7bcd639a9a366b7a7709700e015e9e9fac9a474e7418520d77507ef10fdab4b3R17)